test/unit/spec/MfaVerify_spec.js

```diff
@@ -523,36 +514,54 @@ function (Okta,
 
       });
       Expect.describe('Remember device', function () {
+
+        function getResMfaPolicy(options) {
```

Comment:
can we create a ```setupWithMfaPolicy``` function at the top describe level? That seems more consistent with the general approach we've taken for scenarios like this.
And the calls to it will be something like
```
return setupWithMfaPolicy({ minutes: 0 }).then(function (test) {
```
 or
```
return setupWithMfaPolicy({ minutes: 15, byDefault: true }).then(function (test) {
```

Reply:
Done